torchtext/data/functional.py

```diff
@@ -178,33 +176,31 @@ def read_text_iterator(path, tokenizer):
             yield tokens
 
 
-def create_data_from_iterator(vocab, iterator, include_unk):
-    r"""Create data from an token iterator.
+def create_data_from_iterator(vocab, iterator, removed_tokens=None):
```

Comment:
Make sure to keep track of things that break BC for the release notes. As far as I can tell this does.

Reply:
It's the first time we add `create_data_from_iterator` in `torch.data.functional`.